reactions.json

```diff
@@ -41,7 +41,7 @@
     "AgF": "fluromethane",
     "CH3ONa": "methoxymethane"
   },
-  "chloroethane": {
+  "1-chloroethane": {

```

Comment:
chloroethane should be preferred over 1-chloroethane since it's redundant to write that 1